src/Sylius/Component/Core/Model/ProductVariant.php

```diff
@@ -184,6 +191,27 @@ public function setPrice($price)
     /**
      * {@inheritdoc}
      */
+    public function getOriginalPrice()
+    {
+        return $this->originalPrice;
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
There's no inherited doc for recently added method, these methods should be added to `ProductVariantInterface`.